.github/workflows/check-examples.yml

```diff
@@ -1,13 +1,28 @@
 name: check examples
 
 on:
-  schedule:
-    # 04:00 every weekday morning
-    - cron: '0 4 * * 1-5'
+  pull_request:
+    branches:
+      - master
 
 jobs:
+
+  changes:
+    runs-on: ubuntu-20.04
+    outputs:
+      examples: ${{ steps.filter.outputs.examples }}
+    steps:
+    - uses: dorny/paths-filter@v2
+      id: filter
+      with:
+        filters: |
+          examples:
+            - 'examples/**'
+
   check:
     name: check examples
+    needs: changes
+    if: ${{ needs.changes.outputs.examples == 'true' }}
```

Comment:
Is this different from 'steps.filter.outputs.examples' ? I have not found an example for 'needs.changes'

Reply:
This means the `changes` job is a dependency of this one.

Reply:
Okay, It got clear now and I have seen that in the other PRs.